zingoconfig/src/lib.rs

```diff
@@ -5,7 +5,7 @@ use std::{
     sync::{Arc, RwLock},
 };
 
-use log::{info, LevelFilter};
```

Comment:
This import is needed, to provide access to the `info!` macro called later in the file.   Other than that this PR look good.

Reply:
can this be added conditionally with that macro as well so it's obvious that the import it's needed in such macros?

Reply:
I added a change to address this based on this answer in stackoverflow

https://stackoverflow.com/questions/58167542/how-to-group-imports-in-rust-with-conditional-compilation

Reply:
Ahh..   just read this.

Shall we explicitly qualify instead of using `use` at all?